lib/teaspoon-dev.rb

```diff
@@ -0,0 +1,11 @@
+module Teaspoon
+  DEV_PATH = File.expand_path('../../', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.